Src/Support/Google.Apis.Auth/OAuth2/ExternalAccountCredential.cs

```diff
@@ -335,5 +340,55 @@ public override async Task<bool> RequestAccessTokenAsync(CancellationToken taskC
         /// </summary>
         private protected IGoogleCredential WithUserForDomainWideDelegation(string user) =>
             throw new InvalidOperationException($"{nameof(ExternalAccountCredential)} does not support Domain-Wide Delegation");
+
+        private static readonly IEnumerable<Regex> ValidTokenUrlHostRegexes = new List<Regex>
+        {
+            // sts.xyz123.googleapis.com or exactly sts.googleapis.com
+            new Regex("^sts\\.([^\\.]+\\.)?googleapis\\.com$", RegexOptions.Compiled | RegexOptions.IgnoreCase),
```

Comment:
I assume you're not using verbatim string literals so that we can just copy/paste from Java?

Reply:
Done, it was just left over from copy pasting.